events/api.py

```diff
@@ -134,6 +134,10 @@ def get_authenticated_data_source_and_publisher(request):
         publisher = data_source.owner
         if not publisher:
             raise PermissionDenied(_("Data source doesn't belong to any organization"))
+
+        if publisher.replaced_by:
```

Comment:
This one should not be needed, if we map all api keys to non-replaced organizations?